plugins/fields/list/list.php

```diff
@@ -44,7 +44,17 @@ public function onCustomFieldsBeforePrepareField($context, $item, $field)
 
         $options = $this->getOptionsFromField($field);
 
-        $field->apivalue = [$field->value => $options[$field->value]];
+       if (empty($field->value)) {
```

Comment:
```suggestion
        if (empty($field->value)) {
```
Fix code style errors reported by drone.